### PR TITLE
Fix release instructions, Fix homebrew update

### LINKED ISSUE #686: Set user&email for the commit done by `brew bump-formula-pr`
https://app.circleci.com/pipelines/github/VirtusLab/git-machete/2633/workflows/8acef1bc-8611-4c53-b637-da43c7b33a8f/jobs/12944

image

Just

```shell
git config user.email "gitmachete@virtuslab.com"
git config user.name "Git Machete Bot"
```

### PR DESCRIPTION
**Pull request by amalota**

### CODE REVIEW

**Review by PawelLipski — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -142,7 +142,7 @@ Review fixes should be pushed on separate commits for easier viewing on GitHub (
          git pull origin develop
          git checkout master
          git pull origin master
-         git merge --no-ff develop
+         git merge --ff-only develop
```
Comment (PawelLipski) [inline CONTRIBUTING.md:145]: Pls also update version & release notes

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Release notes
 
+## New in git-machete 3.12.3
+
+- fixed: release instructions in CONTRIBUTING.md are now correct
```
Comment (PawelLipski) [inline RELEASE_NOTES.md:5]:
```suggestion
- fixed: release to Homebrew doesn't crash the deployment process
```

Comment (PawelLipski) [reply RELEASE_NOTES.md:5]: (both are arguably un-interesting to users, but still, let's put the more important one as a filler-up ;p)

Comment (amalota) [reply RELEASE_NOTES.md:5]: Hmm, i'll add both, @PawelLipski ok?

Comment (PawelLipski) [reply RELEASE_NOTES.md:5]: 'kay!

Comment (amalota) [reply RELEASE_NOTES.md:5]: I would say that `fixed: release instructions in CONTRIBUTING.md are now correct` is more important for the user as he doesn't care about deployment

**Review by PawelLipski — APPROVED** [review]

File: ci/homebrew/ci-deploy.sh [hunk 3]
```diff
@@ -19,13 +19,18 @@ sha256=$(
   curl -s https://$pypi_host/pypi/git-machete/"$version"/json \
   | jq --raw-output '.urls | map(select(.packagetype == "sdist")) | .[0].digests.sha256')
 
+git config user.email "gitmachete@virtuslab.com"
+git config user.name "Git Machete Bot"
+
+echo "Bump homebrew formula"
 if [[ $do_push == true ]]; then
-  brew bump-formula-pr --url "$url" --sha256 "$sha256" git-machete
+  brew bump-formula-pr --no-browse --verbose --url "$url" --sha256 "$sha256" git-machete
```
Comment (PawelLipski) [inline ci/homebrew/ci-deploy.sh:27]: Rel: https://github.com/dawidd6/action-homebrew-bump-formula/blob/master/main.rb#L112

**Pull request merged**